crates/sncast/src/helpers/braavos.rs

```diff
@@ -140,3 +143,10 @@ where
             .is_interactive(SignerInteractivityContext::Other)
     }
 }
+
+pub fn assert_non_braavos_account_type(account_type: AccountType) -> Result<(), Error> {
+    if let AccountType::Braavos = account_type {
+        bail!("Integration with Braavos accounts is currently disabled")
```

Comment:
nit:
```suggestion
        bail!("Using Braavos accounts with `sncast` is currently disabled")
```
I would prefer phrasing similar to this, 'integration' feels a bit vague to me